test/BinaryVotingPowerCalculator.unit.t.sol

```diff
@@ -199,6 +272,175 @@ contract Constructor is BinaryVotingPowerCalculatorTest {
   }
 }
 
+contract GetEarningPower is BinaryVotingPowerCalculatorTest {
+  /// @dev Oracle is available && delegate IS eligible → return sqrt(votingPower)
+  function testFuzz_ReturnsVotingPowerIfOracleIsAvailableAndDelegateIsEligible(
+    address _delegate,
+    uint256 _votingPower
+  ) public {
+    _setOracleIsAvailableAndDelegateIsEligible(_delegate, _votingPower);
+
+    uint256 _expectedVotingPower = uint256(Math.sqrt(_votingPower));
+
+    assertEq(calculator.getEarningPower(0, _delegate, address(0)), _expectedVotingPower);
```

Comment:
I think we can fuzz these values in getEarningPower. The test would be a little more robust.

Reply:
Good call